@xen-orchestra/proxy/src/app/mixins/backups/_RemoteAdapter.js

```diff
@@ -30,6 +32,19 @@ export class RemoteAdapter {
     )
   }
 
+  async listAllVmBackups() {
+    const handler = this._handler
+
+    const backups = { __proto__: null }
+    await Promise.all(
+      (await handler.list(BACKUP_DIR, { prependDir: true })).map(async dir => {
+        const vmBackups = await this.listVmBackups(dir)
+        backups[vmBackups[0].vm.uuid] = vmBackups
```

Comment:
Is this [logic](https://github.com/vatesfr/xen-orchestra/blob/dd60d289ff8564f576d398804edf3c852120d12b/packages/xo-server/src/xo-mixins/backups-ng/index.js#L913) implemented?

Reply:
Nope, I will.

Reply:
It's implemented without the remote ID, this is an `xo-server` problematic which should be handled there.